src/AppBanner/index.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from 'react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { BrazeMessage } from '../BrazeMessage';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { knobsData } from '../utils/knobsData';
+
+export default {
+    component: 'AppBanner',
+    title: 'Components/AppBanner',
+    decorators: [withKnobs],
+    parameters: {
+        knobs: {
+            escapeHTML: false, // HTML escaped in order to prevent double-escaping of imgUrl special characters in Storybook
```

Comment:
I feel like there's a missing negative somewhere in this sentence :-) We're telling Storybook not to escape HTML as this results in double escaping/encoding by the time the URL is rendered in the browser.

Reply:
Ah yes, good spot. I've updated it now.